### ISSUE TITLE
Python 3.6

### ISSUE THREAD

**Issue by daniviga** [gem/oq-installers#101]
| ITEM | PR | STATUS |
| ----------------|--------|-----|
| Windows | #102  | open, on hold |
| opt | #102 | open, on hold |
| env | #102 | open, on hold |
| Docker | |blocked by https://github.com/gem/oq-python-rpm/issues/5  |
| VM | na | pending |

**Comment by daniviga** [comment]
VM will be upgraded to Bionic to get use official Python 3.6

**Comment by daniviga** [comment]
Superseded by https://github.com/gem/oq-engine/issues/3864
